### DISCUSSION TITLE
Addition of Player Progression?

### DISCUSSION THREAD

**Discussion by HeelMePlz** [TeKrop/overfast-api#55] [General]
Hii! 
I recently started a little project which I'd hope to use the OverFast API for and was just wondering if there are plans to expand it to include the data that will be displayed on profiles as part of the new Player Progression update coming in Season 6?
Thanks for all the work you already put into this!

**Comment by TeKrop** [comment]
Hello ! First, thank you for your interest un the API 😄 If some new progression data is exposed on Blizzard profile pages, I'll work on this as soon as I have some examples 😀 Even if they display it on another page, as long as this is public, I can work on it 😉

**Reply by HeelMePlz** [reply]
Looks like it appears on the career profiles in-game but no on their website yet 😭 Hopefully something that will come later at least!

**Reply by TeKrop** [reply]
Ah yes indeed 😢 I hope so as well 🤞
